apps/bbsync/tests/test_integration.py

```diff
@@ -832,3 +841,74 @@ def test_flaw_update_multi_cve_restricted(self, auth_client, test_api_uri):
                 format="json",
                 HTTP_BUGZILLA_API_KEY="SECRET",
             )
+
+
+class TestFlawDraftBBSyncIntegration:
+    @pytest.mark.vcr
+    @pytest.mark.enable_signals
+    @pytest.mark.parametrize(
+        "source,cve_id,ext_id",
+        [
+            (Snippet.Source.NVD, "CVE-2000-0025", "CVE-2000-0025"),
+            (Snippet.Source.OSV, "CVE-2000-0026", "GHSA-0006"),
+            (Snippet.Source.OSV, None, "GHSA-0007"),
+        ],
+    )
+    def test_flaw_draft_create(self, source, cve_id, ext_id):
+        """
+        test creating a flaw draft with Bugzilla two-way sync
+        """
+        content = {
+            "cve_id": cve_id,
+            "cvss_scores": [
+                {
+                    "issuer": FlawCVSS.CVSSIssuer.REDHAT,
```

Comment:
Will it work when the issue is not Red Hat? That will most probably be the majority of the cases.

Reply:
Discussed in person.